src/com/hms_networks/americas/sc/realtimedata/RealTimeTagDataPointManager.java

```diff
@@ -0,0 +1,79 @@
+package com.hms_networks.americas.sc.realtimedata;
+
+import java.util.ArrayList;
+
+import com.ewon.ewonitf.EWException;
+import com.ewon.ewonitf.TagControl;
+import com.hms_networks.americas.sc.datapoint.DataPoint;
+import com.hms_networks.americas.sc.logging.Logger;
+
+/**
+ * This class will hold a list of data points for each tag. One instance of the class is made per
+ * tag.

```

Comment:
should have javadoc @author tag.

Reply:
The author on Solution Center files should be:

HMS Networks, MU Americas Solution Center